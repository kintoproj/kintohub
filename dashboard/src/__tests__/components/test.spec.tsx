import React from 'react';
import { shallow } from 'enzyme';

describe('Hello, Enzyme!', () => {
  it('renders', () => {
    const wrapper = shallow(
      <div>
        <h1>Hello, Enzyme!</h1>
      </div>
    );
    expect(wrapper.find('h1').html()).toMatch(/Hello, Enzyme/);
  });

  it('renders snapshots, too', () => {
    const wrapper = shallow(
      <div>
        <h1>Hello, Enzyme!</h1>
      </div>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
