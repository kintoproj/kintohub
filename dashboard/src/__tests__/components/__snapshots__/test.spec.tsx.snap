// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello, Enzyme! renders snapshots, too 1`] = `ShallowWrapper {}`;
